### DISCUSSION TITLE
Avatto WT50 thermostat

### DISCUSSION THREAD

**Discussion by aik** [make-all/tuya-local#739] [Show and tell]
Appears working too, as WT100, just thought worth mentioning. May be something is missing but I have not found it yet :-) This one:

https://www.aliexpress.com/item/1005004988294520.html?spm=a2g0o.productlist.main.5.40065f32yCEwUQ&algo_pvid=f67d9852-8836-431c-bbec-204c32c61f10&algo_exp_id=f67d9852-8836-431c-bbec-204c32c61f10-2&pdp_npi=3%40dis%21AUD%2170.07%2132.76%21%21%21%21%21%4021227f7e16850629138887994d077c%2112000031253318890%21sea%21AU%21711255393&curPageLogUid=UUyZd16vJ9P6

**Comment by aik** [comment]
but now I wonder if it is possible to edit the schedule via HA?

**Reply by make-all** [reply]
No, schedules are an impentatrible blob of data that you would need to reverse engineer to find its format, then create a custom card  in HA to handle the different subfields in a meaningful way.

This is typically the extent of the documentation you will find for a schedule field:

```
      {
        "code": "schedule",
        "dp_id": 103,
        "type": "Raw",
        "values": "{}"
      },
```

The type "Raw" clues you in that this is a blob of binary data, probably encoded as base64, or possibly hex, but there is no information about what fields you will find in that binary blob.
